src/app/core/services/search-registration/search-registration.service.ts

```diff
@@ -50,7 +51,7 @@ export class SearchResult<TViewModel> {
   }
 }
 
-export class PagedSearchResult<TViewModel> {
+export class PagedSearchResult<TViewModel extends Pick<RegistrationViewModel, 'RegId'>> {
```

Comment:
Går det an å deklarere Pick<RegistrationViewModel, 'RegId'> som en egen type for lesbarhetens skyld?
Ingen stor sak altså!-)

Reply:
Jepp, det ble bedre 👍